File: Spartane.Web/Uploads/Scripts/Detalle_de_Seguimiento.js
$(document).ready(function () {
    var inpuElementArray = [
        { "inputId": "Clave", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Especialista", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Solicitante", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Requerido", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Archivo", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Observaciones", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Cumplimiento", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Tipo_de_Seguimiento", "inputType": "select", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Cantidad_del_Monto", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }
        ,{ "inputId": "Fecha_de_Cumplimiento", "inputType": "text", "IsRequired": true, "IsVisible": true, "IsReadOnly": false, "DefaultValue": "", "HelpText": "" }

    ];
    setInputEntityAttributes(inpuElementArray, "#Detalle_de_Seguimiento", 1);
});

File: Spartane.Core/Domain/Detalle_de_Seguimiento/Detalle_de_Seguimiento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Spartane.Core.Domain.Tipo_de_Seguimiento;
using Spartane.Core.Domain.Solicitud;

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Web.Mvc;

namespace Spartane.Core.Domain.Detalle_de_Seguimiento
{
    /// <summary>
    /// Detalle_de_Seguimiento table
    /// </summary>
    public class Detalle_de_Seguimiento: BaseEntity
    {
        public int Clave { get; set; }
        public string Especialista { get; set; }
        public string Solicitante { get; set; }
        public string Requerido { get; set; }
        public short? Archivo { get; set; }
        public string Observaciones { get; set; }
        public string Cumplimiento { get; set; }
        public int? Tipo_de_Seguimiento { get; set; }
        public decimal? Cantidad_del_Monto { get; set; }
        public DateTime? Fecha_de_Cumplimiento { get; set; }
        public int? Solicitud { get; set; }

        [ForeignKey("Tipo_de_Seguimiento")]
        public virtual Spartane.Core.Domain.Tipo_de_Seguimiento.Tipo_de_Seguimiento Tipo_de_Seguimiento_Tipo_de_Seguimiento { get; set; }
        [ForeignKey("Solicitud")]
        public virtual Spartane.Core.Domain.Solicitud.Solicitud Solicitud_Solicitud { get; set; }

    }
	
	public class Detalle_de_Seguimiento_Datos_Generales
    {
                public int Clave { get; set; }
        public string Especialista { get; set; }
        public string Solicitante { get; set; }
        public string Requerido { get; set; }
        public short? Archivo { get; set; }
        public string Observaciones { get; set; }
        public string Cumplimiento { get; set; }
        public int? Tipo_de_Seguimiento { get; set; }
        public decimal? Cantidad_del_Monto { get; set; }
        public DateTime? Fecha_de_Cumplimiento { get; set; }
        public int? Solicitud { get; set; }

		        [ForeignKey("Tipo_de_Seguimiento")]
        public virtual Spartane.Core.Domain.Tipo_de_Seguimiento.Tipo_de_Seguimiento Tipo_de_Seguimiento_Tipo_de_Seguimiento { get; set; }
        [ForeignKey("Solicitud")]
        public virtual Spartane.Core.Domain.Solicitud.Solicitud Solicitud_Solicitud { get; set; }

    }


}

